modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -16,6 +16,135 @@
+
+* The following services have been added:
+
+** xref:next@ocis:ROOT:{s-path}/activitylog.adoc[activitylog] +
+It is responsible for storing events (activities) per resource. +
+Note that the implementation of the `activitylog` service is a breaking change. For details see xref:breaking-changes[Breaking Changes] section.
+
+** xref:next@ocis:ROOT:{s-path}/auth-app.adoc[auth-app] +
+It provides authentication for 3rd party apps.
+
+** xref:next@ocis:ROOT:{s-path}/collaboration.adoc[collaboration] +
+It connects with web office document servers such as Collabora, ONLYOFFICE or Microsoft using the WOPI protocol. Note that the `collaboration` service replaces the external https://github.com/cs3org/wopiserver[cs3org/wopi] server used before. If you have used document servers with the external wopiserver, we highly recommend using the collaboration service which is a breaking change. For details see xref:breaking-changes[Breaking Changes] section.
+
+* Content Security Policy: +
+When using external services like an IDP, web office document servers or web apps (like provided in our docker compose deployment examples), a Content Security Policy (CSP) has been implemented to secure the environment. For details on web office/apps see the next list item. For details on CSP see xref:breaking-changes[Breaking Changes] section.
+
+* Fully customisable deployment examples ready for production use including web office, based on docker compose. See the
+xref:next@ocis:ROOT:depl-examples/ubuntu-compose/ubuntu-compose-prod.adoc[Local Production Setup] and xref:next@ocis:ROOT:depl-examples/ubuntu-compose/ubuntu-compose-hetzner.adoc[Deployment on Hetzner]. +
+Note that with these examples, you can easily enable additional web apps like `draw-io`, `external-sites`, `json-viewer`, `unzip`, `progressbars` and others. +
+Though provided via the compose example setup, no support can be given for these web apps.
+
+* Additonal external web apps can be configured _manually_ to the deployment examples.
+** https://github.com/owncloud/web-app-dicom-viewer[Dicom Viewer] +
+The Dicom Viewer Web Extension offers an easy way to preview medical images of DICOM standard (.dcm) such as Ultrasound, X-ray, CT scan, MRI, including all their corresponding meta data. The app offers image manipulation operations such as zoom in and out, rotation, flipping, colour inversion on the image preview.
+
+** https://github.com/saw-jan/web-app-3dmodel-viewer[3D Model Viewer] +
+The 3D model viewer Web Extension is an extension for ownCloud Web which offers preview of .glb, .stl, .fbx, and .obj files. The app allows to zoom and rotate the model, to navigate between model files, plus it supports full screen view.
+
+** http://github.com/JankariTech/web-app-presentation-viewer[Presentation Viewer] +
+The Presentation Viewer Web Extension allows to render markdown slide presentations from .md files with reveal.js. It can be used both for preview and for sharing the slides using public links.
```

Comment:
```suggestion
The Presentation Viewer Web Extension allows rendering markdown slide presentations from .md files with reveal.js. It can be used both for preview and for sharing the slides using public links.
```